src/main/java/net/sf/mzmine/modules/visualization/molstructure/MolStructureViewer.java

```diff
@@ -238,4 +238,8 @@ private void load3DStructure(URL url) {
     }
 
   }
+
+  public JSplitPane getSplitPane() {

```

Comment:
This function is added but it doesn't seem you actually use it anywhere?

Reply:
oh yes, this was a leftover from another approach. I have removed it